src/components/forms/AddressInput/index.jsx

```diff
@@ -63,36 +63,6 @@ const AddressInput = ({
         }
       }}
     </OnChange>
-    {/* onBlur - didn't work because of the complex validation
-    (if you submit before it gets the address, breaks everything) */}
```

Comment:
I understand that the intention of leaving the message is for future reference.

But I prefer we discuss this in the PR and completely remove this whole unused block.

Reply:
it's already removed, isn't it?

Reply:
:facepalm: